keyboards/framework/readme.md

```diff
@@ -0,0 +1,44 @@
+# Framework Laptop 16 Keyboard
+
+Keyboard input modules for the Framework Laptop 16.
+
+* Keyboard Maintainer: [Daniel Schaefer](https://github.com/JohnAZoidberg)
+* Hardware Supported: Framework Laptop 16 Keyboard, Numpad and Macropad
+* Hardware Availability: Soon available at https://frame.work/marketplace
+
+## Variants
+
+There are 5 different hardware variants with different number of keys and
+backlight options.
+
+| Name     | Size     | Backlight  | Keys |
+| -------- | -------- | ---------- | ---- |
+| ansi     | Keyboard | RGB/White  |   78 |
```

Comment:
Are these RGB and White backlight versions actually different PCBs?  You may need to create separate firmwares for them, so that the RGB code won't run into I2C timeouts if it attempts to access nonexistent chips on the simple backlight version.

Reply:
It is a different PCB yes. But it would be nice to keep the the firmware the same.
In a future PR where I add the RGB variant, I can make it disable RGB once it finds the LED controller is not there.